examples/dt/nova/nova02beta/service-values.yaml

```diff
@@ -0,0 +1,58 @@
+# local-config: referenced, but not emitted by kustomize
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preserveJobs: false
+  neutron:
+    customServiceConfig: |
+      [ml2]
+      mechanism_drivers = ovn,sriovnicswitch
+      [ml2_type_vlan]
+      network_vlan_ranges = sriov-phy4
+  ovn:
+    ovnController:
+      nicMappings:
+        datacentre: ospbr
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
+      [glance_store]
+      default_backend = default_backend
+      [default_backend]
+      swift_store_create_container_on_put = True
+      swift_store_auth_version = 3
+      swift_store_auth_address = {{ .KeystoneInternalURL }}
+      swift_store_endpoint_type = internalURL
+      swift_store_user = service:glance
+      swift_store_key = {{ .ServicePassword }}
+    default:
+      replicas: 1
+  swift:
+    enabled: true
+  nova:
+    apiServiceTemplate:
+      customServiceConfig: |
+        [pci]
+        alias = { "vendor_id":"8086", "product_id":"154d", "device_type":"type-PCI", "name":"a1" }
+        [filter_scheduler]
+        pci_in_placement = True
+    cell0:
+      conductorServiceTemplate:
+        customServiceConfig: |
+          [filter_scheduler]
+          pci_in_placement = True
+    cell1:
+      conductorServiceTemplate:
+        customServiceConfig: |
+          [filter_scheduler]
+          pci_in_placement = True
+    schedulerServiceTemplate:
+      customServiceConfig: |
+        [filter_scheduler]
+        enabled_filters = AvailabilityZoneFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,ServerGroupAntiAffinityFilter,ServerGroupAffinityFilter,PciPassthroughFilter,NUMATopologyFilter,AggregateInstanceExtraSpecsFilter
```

Comment:
why is this being customized?
we default to 

AggregateInstanceExtraSpecsFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,ServerGroupAntiAffinityFilter,ServerGroupAffinityFilter,SameHostFilter,DifferentHostFilter,PciPassthroughFilter,NUMATopologyFilter

https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf#L89

the order of the filter affect performace so we order them that way based on the cost of the filters.

the AvailabilityZoneFilter is deprecated and not requried in 18 we replaced it with placement.

all the filster you are enabling are enabeld in the default set in the correct order so i woudl jsut drop this form the change.

Reply:
Thanks, this was copied over from an older DT.  I've removed it completely.